src/project/git/GitCommandGitProject.ts

```diff
@@ -48,11 +53,15 @@ export class GitCommandGitProject extends NodeFsLocalProject implements GitProje
      * @param {RepoRef} id
      * @param {string} baseDir
      * @param {ProjectOperationCredentials} credentials
+     * @param release call this when you're done with the project. make its filesystem resources available to others.
+     * @param provenance optional; for debugging, describe how this was constructed
```

Comment:
provenance is for debugging. Optional, but super useful, highly recommend. It helped me figure things out.

Reply:
Nice